### ISSUE TITLE
logo

### ISSUE THREAD

**Issue by ihustle662** [marcostom32/qb-multicharacter#18]
is it possible to replace the qb logo at the start and if so how big should the file be

**Comment by Bama94** [comment]
> is it possible to replace the qb logo at the start and if so how big should the file be

Yes, what I do is go into qb-multicharacter>html and open up qb-pixel.png with a image editor like photoshop and then I import my logo on top of it and make it the same height as the qb logo, then remove the qb logo and save it. then your good to go!
